app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -436,7 +436,8 @@ data class Patient(
 
     @Query("""
         DELETE FROM Patient
-        WHERE deletedAt IS NOT NULL AND retainUntil > :now
+        WHERE deletedAt IS NOT NULL AND retainUntil < :now 
```

Comment:
So, unless the user is soft deleted on the device. We don't remove them, even if the retention time is completed?

For example, let's say a "Patient X" is synced by the online patient lookup with a 1 day retention time. Some measurements were updated, and then synced. Once the retention period is completed, I don't think this query will remove it since the `deleatedAt` for "Patient X" is null. Please, confirm the behaviour.

Reply:
The `deletedAt` check should only be used for soft deletes. This property should not factor into records fetched for online lookup.

Reply:
Also, `retainUntil` will be `NULL` for most patients. Have we verified how this query behaves in this case?

Reply:
^ Similar feedback for the other comments as well.